server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -447,42 +412,55 @@ private DivTag renderIndexPageLink(AccountModel applicant, Messages messages) {
+    return new ATag()
+        .withId("account-settings-link")
+        .withHref(
+            baseUrl + controllers.ti.routes.TrustedIntermediaryController.accountSettings().url())
+        .withText(messages.at(MessageKey.HEADER_ACCT_SETTING.getKeyName()));
   }
 
-  private TdTag renderEmailCell(AccountModel ti) {
-    String emailField = ti.getEmailAddress();
-    if (Strings.isNullOrEmpty(emailField)) {
-      emailField = "(no email address)";
-    }
-    return td(emailField).withClasses(ReferenceClasses.BT_EMAIL);
+  private static ATag renderClientListLink(Messages messages, String baseUrl) {
+    return new ATag()
+        .withId("client-list-link")
+        .withHref(
+            baseUrl
+                + controllers.ti.routes.TrustedIntermediaryController.dashboard(
+                        /* nameQuery= */ Optional.empty(),
+                        /* dayQuery= */ Optional.empty(),
+                        /* monthQuery= */ Optional.empty(),
+                        /* yearQuery= */ Optional.empty(),
+                        /* page= */ Optional.of(1))
+                    .url())
+        .withText(messages.at(MessageKey.HEADER_CLIENT_LIST.getKeyName()));
   }
 
-  private TdTag renderStatusCell(AccountModel ti) {
-    String accountStatus = "OK";
-    if (ti.ownedApplicantIds().isEmpty()) {
-      accountStatus = "Not yet signed in.";
-    }
-    return td(accountStatus);
+  public enum TabType {
+    CLIENT_LIST,
+    ACCOUNT_SETTINGS
   }
 
-  private TheadTag renderGroupTableHeader(Messages messages) {
-    return thead(
-        tr().with(
-                th(messages.at(MessageKey.NAME_LABEL.getKeyName()))
-                    .withScope("col")
-                    .withData("testid", "org-members-name")
-                    .withClass("w-1/3"))
-            .with(
-                th(messages.at(MessageKey.EMAIL_LABEL.getKeyName()))
-                    .withScope("col")
-                    .withData("testid", "org-members-email")
-                    .withClass("w-2/5"))
-            .with(
-                th(messages.at(MessageKey.ACCT_STATUS_LABEL.getKeyName()))
-                    .withScope("col")
-                    .withData("testid", "org-members-status")
-                    .withClass("w-1/5")));
+  public static NavTag renderTabButtons(Messages messages, String baseUrl, TabType currentTab) {
+    boolean isClientList = currentTab == TabType.CLIENT_LIST;
+    boolean isAccountSettings = currentTab == TabType.ACCOUNT_SETTINGS;
+    return nav(
+            renderClientListLink(messages, baseUrl)
+                .withClasses(
+                    "mr-10",
+                    "py-4",
+                    iffElse(isClientList, "border-b-4", "hover:border-b-4"),
+                    iffElse(isClientList, "border-blue-600", "hover:border-blue-600"),
+                    iffElse(isClientList, "text-bold", "hover:text-blue-600"))
```

Comment:
**suggestion:** We could consider factoring this shared logic into a separate function, since its similar / the same below?

Reply:
I used your idea, Rocky, and created a couple of `BaseStyles`.